website/docs/developer/deployment-guide/deploying-cumulus-dashboard-from-mac.md

```diff
@@ -0,0 +1,117 @@
+    ```bash
+    ssh -vvv -p 6868 -N -D 1337 -i ~/.ssh/<YOUR_AWS_PRIVATE_KEY.pem> -o "UserKnownHostsFile=/dev/null" -o "StrictHostKeyChecking=no" ec2-user@127.0.0.1
+    ```
+
+  Replace `<YOUR_AWS_PRIVATE_KEY.pem>` with yours.
+
+ :::note
+ Port -p 6868 should match the localPortNumber=6868 port number in your AWS SSM session.
+ :::
+
+ 4. The next step is to deploy the dashboard. Under `cumulus-dashboard` directory, run 
+  
+   ```bash
+   npm install
+   ```
+
+   Once that is done, run the following command by replacing `<YOUR_API_ENDPOINT>` with your Cumulus API endpoint found from terraform output.
+
+    ```bash
+   ENABLE_RECOVERY=true APIROOT=<YOUR_API_ENDPOINT> npm run serve
+   ```
+
+ 5. Once the website is running, go to Firefox and put in `http://localhost:3000/` to access the dashboard.   
+ 
+   :::important
+   Make sure the SSM and SSH sessions are both running.
+   :::important
+
+
+## Deploying the dashboard with launchpad authentication
+
+A workflow request followed by a NAMS request has to be created and approved first before the launchpad authentication can be used. Check https://idmax.nasa.gov/nams/asset/255115
+
+:::note
+There might be some issues while creating the workflow request. Contact ORCA team while creating the NAMS request.
+
+:::
+
+Once the workflow and NAMS request are approved, proceed to the following steps.
+
+1. Go to your AWS S3 console and search for `<prefix>-internal` bucket. Under `<prefix>-internal/<prefix>/` directory, create a new folder `crypto`.
+   Upload the two files `launchpadMetadata.xml` and `launchpad.pfx` under `<prefix>-internal/<prefix>/crypto`. 
+   Contact ORCA team if you do not have access to those files.
+2. Update your `cumulus-tf/terraform.tfvars` file to include the following variables:
+    
+   ```
+    cmr_oauth_provider = "launchpad"
+    oauth_provider   = "launchpad"
+    launchpad_api = "https://api.launchpad.nasa.gov/icam/api/sm/v1"
+    # using the standard path, which is <prefix>/crypto/launchpad.pfx in the system bucket
+    # launchpad_certificate = "certificate"
+    launchpad_passphrase = "35kbMfH25Y4JjCBfoj7XBig!"
+    oauth_user_group = "LPDAAC-cumulus-operator-uat"
+    # see Wiki: https://wiki.earthdata.nasa.gov/display/CUMULUS/Cumulus+SAML+Launchpad+Integration
+    saml_entity_id                  = "https://<YOUR_API_ENDPOINT>"
+    saml_assertion_consumer_service = "https://<YOUR_API_ENDPOINT>/saml/auth"
+    saml_idp_login                  = "https://auth.launchpad.nasa.gov/affwebservices/public/saml2sso"
+    saml_launchpad_metadata_url     = "https://auth.launchpad.nasa.gov/unauth/metadata/launchpad.idp.xml"
+
+   ```
+
+   Replace `<YOUR_API_ENDPOINT>` with your Cumulus API endpoint found from terraform output.
+
+3. Rerun `terraform apply` to deploy the updates under `cumulus-tf`.
+
+
+4. Follow all the steps under `Setting up proxy and tunneling to locally run the dashboard` with the exception of step 4. In step 4 above,
```

Comment:
Wrong name. Might want to link to an anchor as well.

Reply:
You still want them to run the old step 4, including the `npm install`, right? Better wording might be "... the dashboard` until the npm run serve command in step 4.

Reply:
yes correct. I will update the wording